<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>8</number>
      </property>
      <property name="topMargin">
       <number>8</number>
      </property>
      <property name="rightMargin">
       <number>8</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>输入</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QTextEdit" name="inputTextEdit"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>颜色</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="3" column="1">
            <widget class="QLabel" name="gridColorLabel1">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>田字格横竖</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="11" column="1">
            <widget class="QLineEdit" name="gridDashLenInput2">
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="placeholderText">
              <string>3</string>
             </property>
            </widget>
           </item>
           <item row="12" column="3">
            <widget class="QCheckBox" name="borderVisibility">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="newStrokeColorLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>当前笔画</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
             <property name="indent">
              <number>-1</number>
             </property>
            </widget>
           </item>
           <item row="9" column="1">
            <widget class="QLabel" name="gridDashLenLabel2">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>对角虚线长度</string>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="QLineEdit" name="gridWidthInput1">
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="placeholderText">
              <string>0.5</string>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QCheckBox" name="gridVisibility1">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="12" column="1">
            <widget class="QLabel" name="borderColorLabel">
             <property name="text">
              <string>边框</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLineEdit" name="gridDashLenInput1">
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="placeholderText">
              <string>3</string>
             </property>
            </widget>
           </item>
           <item row="5" column="2">
            <widget class="QLabel" name="gridDashSpaceLabel1">
             <property name="text">
              <string>横竖虚线间距</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="doneStrokeColorLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>已绘制笔画</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="13" column="2">
            <widget class="QLineEdit" name="borderWidthInput">
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="placeholderText">
              <string>2</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLineEdit" name="newStrokeColorInput">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="inputMask">
              <string/>
             </property>
             <property name="maxLength">
              <number>9</number>
             </property>
             <property name="placeholderText">
              <string>#FFFFFF</string>
             </property>
            </widget>
           </item>
           <item row="7" column="2">
            <widget class="QLineEdit" name="gridColorInput2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>20</height>
              </size>
             </property>
             <property name="maxLength">
              <number>9</number>
             </property>
             <property name="placeholderText">
              <string>#FFFFFF</string>
             </property>
            </widget>
           </item>
           <item row="8" column="2">
            <widget class="QLineEdit" name="gridWidthInput2">
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="placeholderText">
              <string>0.5</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="undoneStrokeColorLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>未绘制笔画</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLabel" name="gridDashLenLabel1">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>横竖虚线长度</string>
             </property>
            </widget>
           </item>
           <item row="13" column="1">
            <widget class="QLabel" name="borderWidthLabel">
             <property name="text">
              <string>边框粗细</string>
             </property>
            </widget>
           </item>
           <item row="11" column="2">
            <widget class="QLineEdit" name="gridDashSpaceInput2">
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="placeholderText">
              <string>3</string>
             </property>
            </widget>
           </item>
           <item row="9" column="2">
            <widget class="QLabel" name="gridDashSpaceLabel2">
             <property name="text">
              <string>对角虚线间距</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLineEdit" name="doneStrokeColorInput">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="inputMask">
              <string/>
             </property>
             <property name="maxLength">
              <number>9</number>
             </property>
             <property name="placeholderText">
              <string>#FFFFFF</string>
             </property>
            </widget>
           </item>
           <item row="7" column="3">
            <widget class="QCheckBox" name="gridVisibility2">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLineEdit" name="undoneStrokeColorInput">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>20</height>
              </size>
             </property>
             <property name="inputMask">
              <string/>
             </property>
             <property name="maxLength">
              <number>9</number>
             </property>
             <property name="placeholderText">
              <string>#FFFFFF</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QLabel" name="gridWidthLabel2">
             <property name="text">
              <string>对角虚线粗细</string>
             </property>
            </widget>
           </item>
           <item row="6" column="2">
            <widget class="QLineEdit" name="gridDashSpaceInput1">
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="placeholderText">
              <string>3</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="gridWidthLabel1">
             <property name="text">
              <string>横竖虚线粗细</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QLabel" name="gridColorLabel2">
             <property name="text">
              <string>田字格对角</string>
             </property>
            </widget>
           </item>
           <item row="12" column="2">
            <widget class="QLineEdit" name="borderColorInput">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="maxLength">
              <number>9</number>
             </property>
             <property name="placeholderText">
              <string>#FFFFFF</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QLineEdit" name="gridColorInput1">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="inputMask">
              <string/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="maxLength">
              <number>9</number>
             </property>
             <property name="placeholderText">
              <string>#FFFFFF</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>预览</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QComboBox" name="curCharComboBox"/>
           </item>
           <item>
            <widget class="QSplitter" name="splitter_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <widget class="QGraphicsView" name="strokePreview">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="sizeAdjustPolicy">
               <enum>QAbstractScrollArea::AdjustIgnored</enum>
              </property>
             </widget>
             <widget class="QWidget" name="horizontalLayoutWidget_2">
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QSplitter" name="splitter">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <widget class="QComboBox" name="curStrokeComboBox">
                  <property name="maximumSize">
                   <size>
                    <width>50</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="editable">
                   <bool>true</bool>
                  </property>
                 </widget>
                 <widget class="QLabel" name="curStrokeSplitterLabel">
                  <property name="maximumSize">
                   <size>
                    <width>10</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>/</string>
                  </property>
                 </widget>
                 <widget class="QLabel" name="totalStrokeLabel">
                  <property name="maximumSize">
                   <size>
                    <width>50</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>-</string>
                  </property>
                 </widget>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="updatePreviewBtn">
               <property name="text">
                <string>预览</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="outputDirLabel">
          <property name="text">
           <string>保存位置</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="outputDirInput"/>
        </item>
        <item>
         <widget class="QPushButton" name="outputDirSelectBtn">
          <property name="text">
           <string>选择...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QProgressBar" name="progressBar">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="startBtn">
            <property name="text">
             <string>开始</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="closeBtn">
            <property name="text">
             <string>关闭</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
